lib/src/layer/polygon_layer/label.dart

```diff
@@ -6,42 +6,54 @@ import 'package:flutter_map/src/layer/polygon_layer/polygon_layer.dart';
 import 'package:latlong2/latlong.dart';
 import 'package:polylabel/polylabel.dart';
 
+class PolygonBounds {
+  final Offset min;
+  final Offset max;
+
+  const PolygonBounds(this.min, this.max);
+}
```

Comment:
For a small class like this without methods, I'd usually prefer a `Record`: `({Offset min, Offset max})`.

Reply:
Works for me. Changed